packages/inspection-capture-web/test/PhotoCapture/PhotoCapture.test.tsx

```diff
@@ -49,6 +55,10 @@ jest.mock('../../src/PhotoCapture/hooks', () => ({
     },
   })),
   useTracking: jest.fn(),
+  usePhotoCaptureTutorial: jest.fn(() => ({
+    tutorialView: 'welcome',
+    handleTutorialView: jest.fn(),
+  })),
```

Comment:
Update the mock of this hook to return the proper variable names. Also, it makes me think : if your mock is not returning the proper variable names, but your tests still work properly, maybe your tests aren't precise enough? I haven't taken a look at your tests in depth so I'm just mentioning this here but maybe they're good